CMakeLists.txt

```diff
@@ -495,20 +495,21 @@ if(UHDR_ENABLE_INSTALL)
   set(UHDR_TARGET_NAME uhdr)
   add_library(${UHDR_TARGET_NAME} SHARED)
   add_dependencies(${UHDR_TARGET_NAME} ${UHDR_CORE_LIB_NAME})
+  target_link_libraries(${UHDR_TARGET_NAME} PRIVATE ${JPEG_LIBRARIES})
   set_target_properties(${UHDR_TARGET_NAME} PROPERTIES PUBLIC_HEADER ultrahdr_api.h)
   combine_static_libs(${UHDR_CORE_LIB_NAME} ${UHDR_TARGET_NAME})
 
-  if(NOT MSVC)
+  if(NOT(WIN32 OR XCODE))
     include(GNUInstallDirs)
 
     # pkg-config: libuhdr.pc
     configure_file("${CMAKE_CURRENT_SOURCE_DIR}/libuhdr.pc.template"
                    "${CMAKE_CURRENT_BINARY_DIR}/libuhdr.pc" @ONLY NEWLINE_STYLE UNIX)
     install(FILES "${CMAKE_CURRENT_BINARY_DIR}/libuhdr.pc"
-            DESTINATION "${CMAKE_INSTALL_PREFIX}/lib/${CMAKE_LIBRARY_ARCHITECTURE}/pkgconfig")
+            DESTINATION "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/pkgconfig")
     install(TARGETS ${UHDR_TARGET_NAME}
-            DESTINATION "${CMAKE_INSTALL_PREFIX}/lib/${CMAKE_LIBRARY_ARCHITECTURE}/"
-            PUBLIC_HEADER DESTINATION "${CMAKE_INSTALL_PREFIX}/include")
+            DESTINATION "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"
+            PUBLIC_HEADER DESTINATION "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_INCLUDEDIR}")
```

Comment:
One last thing: `${CMAKE_INSTALL_PREFIX}` is implied and not necessary for the install() command. Moreover, on some rare systems like NixOS `${CMAKE_INSTALL_LIBDIR}` and others translate to absolute paths so this can actually lead to an error.